mempool/client.go

```diff
@@ -0,0 +1,52 @@
+package mempool
+
+import (
+	"unsafe"
+
+	mempoolproto "github.com/tendermint/tendermint/proto/tendermint/mempool"
+)
+
+type Client struct {
+	mp Mempool
+}
+
+func NewMempoolClient(memInt int64) *Client {
+	ptrVal := uintptr(memInt)
+	ptr := unsafe.Pointer(ptrVal)
+	memPtr := (*Mempool)(ptr)
+	cli := &Client{
+		mp: *memPtr,
+	}
+	return cli
+}
+
+func (cli *Client) GetNextTransaction(
+	req *mempoolproto.GetNextTransactionRequest) (*mempoolproto.GetNextTransactionResponse, error) {
```

Comment:
we don't necessarily need to follow the GRPC API. 

`GetNextTransaction` can simply just return what its mempool client returns